app/dashboard/templates/profiles/profile.html

```diff
@@ -19,7 +19,7 @@
         <div class="row">
           <div class="col-12 col-lg-4 profile-header__main-infos">
             <div class="clearfix">
-              <img class="profile-header__avatar mb-1 mr-4" src="{% if profile.avatar and profile.avatar.avatar_url %}{{ profile.avatar.avatar_url }}{% else %}{{ profile.avatar_url }}{% endif %}">
+              <img class="profile-header__avatar mb-1 mr-4" src="{% if profile.avatar and profile.avatar.avatar_url %}{{ profile.avatar.avatar_url }}{% else %}{{ profile.avatar_url }}{% endif %}" />
```

Comment:
@kuhnchris `img` tags don't require closure xD

Reply:
it depends, are we "just" HTML or XHTML? ;-)
But basically - you are correct.
https://www.w3.org/TR/html5/syntax.html#void-elements